packages/design-system/src/hooks/use-defaults/types.ts

```diff
@@ -122,6 +129,18 @@ export type DefaultsContext = DefaultsOf<{
   // biome-ignore lint/style/useNamingConvention: component name should be PascalCase
   Select: SelectProps<object>;
   // biome-ignore lint/style/useNamingConvention: component name should be PascalCase
+  Slider: SliderProps;
+  // biome-ignore lint/style/useNamingConvention: component name should be PascalCase
```

Comment:
Dear lord, who did this Biome commenting? Can you create an issue to convert to range suppression https://github.com/biomejs/biome/issues/4569 please?

Reply:
that was me!

see PR/discussion here: https://github.com/gohypergiant/standard-toolkit/pull/158#discussion_r1904570169

we can create an issue to use the range suppression, but it looks like [it won't be available until 2.0 is released](https://github.com/biomejs/biome/pull/4649#issuecomment-2524003559)